src/partials/install_instructions.html

```diff
@@ -1,131 +1,94 @@
-<div class="mt-8 md:mt-14 w-full max-w-[600px] flex flex-col items-center">
-  <div role="tablist" class="tabs w-full justify-between" data-el="tabs">
-    <a
-      id="link-install-flyio"
-      class="tab active"
-      href="#"
-      role="tab"
-      aria-controls="install-flyio"
-      aria-selected="true"
-      data-tab-target="#install-flyio"
-    >
-      Launch on Fly.io
-    </a>
-    <a
-      id="link-install-docker"
-      class="tab"
-      href="#"
-      role="tab"
-      aria-controls="install-docker"
-      data-tab-target="#install-docker"
-      >Launch using Docker</a
-    >
-    <a
-      id="link-install-elixir"
-      class="tab"
-      href="#"
-      role="tab"
-      aria-controls="install-elixir"
-      data-tab-target="#install-elixir"
-      >Launch using Elixir</a
-    >
-  </div>
+<div class="m-auto max-w-7x p-4 mt-5 md:max-w-[932px]">
   <div class="mt-4 w-full p-4 rounded-2xl bg-white">
     <img src="/images/window_buttons.png" alt="" width="36" height="8" />
-    <!-- Tab contents -->
-
-    <!-- * Fly.io instructions -->
     <div
-      id="install-flyio"
-      role="tabpanel"
-      aria-labelledby="link-install-flyio"
-      data-el="tab-content"
-      data-tab-active
-      class="my-12 flex flex-col items-center justify-center"
+      class="px-1 py-10 flex maxsm:flex-col flex-wrap md:flex-nowrap justify-center"
     >
-      <a
-        class="px-5 py-3 flex space-x-3 items-center rounded-xl bg-[#8b5cf6] hover:bg-[#7c3aed] text-white"
-        href="https://fly.io/launch/livebook"
-      >
-        <img src="/images/flyio_logo_white.svg" alt="" width="32" height="27" />
-        <span class="font-semibold -mb-0.5">Launch on Fly</span>
-      </a>
-    </div>
+      <!-- fly.io -->
+      <div class="flex flex-col md:items-start md:border-r md:pr-4">
+        <div class="flex flex-nowrap min-w-[270px]">
+          <img class="self-center" src="/images/icon_flyio.png" />
```

Comment:
Please download the .svg version from Figma, this way it will look much more sharp :D Same for other icons.

Reply:
Done!